docs/site/Deploying_to_ibm_cloud_kubernetes.md

```diff
@@ -0,0 +1,707 @@
+
+Wait until it completes.
+
+In your IBM Cloud account, you can view your images [here](https://cloud.ibm.com/containers-kubernetes/registry/main/images).
+
+
+## Step 15: Point to your Kubernetes Cluster
+
+In a browser, log into your IBM Cloud account, and navigate to Kubernetes > Clusters.
+
+
+![](./imgs/lb4_k8s_ibm_cloud_kubernetes_cluster_1.png)
+
+I am choosing my cluster ````dremondOne```` in Dallas .
+
+
+Select the "Access" tab of your cluster and follow the instructions.
+
+![](./imgs/lb4_k8s_ibm_cloud_cluster_access_1.png)
+
+
+We already logged into the IBM Cloud in an earlier step, so we only need
+to point to the cluster.
+
+````
+ibmcloud cs region-set us-south
+````
+
+````
+ibmcloud cs cluster-config <Cluster Name>
+````
+
+My cluster name is "dremondOne" so I see this output:
+
+````
+OK
+The configuration for dremondOne was downloaded successfully. Export environment variables to start using Kubernetes.
+
+export KUBECONFIG=/Users/dremond/.bluemix/plugins/container-service/clusters/dremondOne/kube-config-hou02-dremondOne.yml
+````
+
+Take the entire "export" line above, and paste it into your command window.
+
+
+Now you should be able to perform Kubernetes commands like:
+
+````
+kubectl get nodes
+````
+
+You will see output like this:
+
+````
+NAME           STATUS    ROLES     AGE       VERSION
+10.76.193.58   Ready     <none>    13d       v1.10.8+IKS
+````
+
+Ok, so now we are ready to deploy our Loopback4 application to Kubernetes!
+
+
+## Step 16: Deploy your Loopback4 application to Kubernetes
+
+
+### Create a deployment
+
+Create a deployment named : lb4-simple-web-app-deployment ; using the image we placed in the container registry.
+
+````
+kubectl run lb4-simple-web-app-deployment --image=registry.ng.bluemix.net/dremond/loopback4_webapp_repo:1
+````
+
+### Verify that the pods are running
+
+```
+kubectl get pods
+```
+
+You should see
+
+````
+NAME                                             READY     STATUS             RESTARTS   AGE
+lb4-simple-web-app-deployment-5bfcb546d8-r7cs4   1/1       Running            0          7m
+````
+
+A status of "Running" is a good sign. If you have anything other than this, then there may be something wrong with your docker image , or it may have vulnerability issues you need to address.
+
+To see the logs of your pod, type:
+
+````
+kubectl logs lb4-simple-web-app-deployment-5bfcb546d8-r7cs4
+````
+
+and you will see something like this:
+
+
+````
+
+> deploy-test@1.0.0 start /
+> node .
+
+Server is running at http://127.0.0.1:8080
+Try http://127.0.0.1:8080/ping
+````
+
+### Create a service
+
+Expose your deployment with a service named : lb4-simple-web-app-service
+
+````
+kubectl expose deployment/lb4-simple-web-app-deployment --type=NodePort --port=8080 --name=lb4-simple-web-app-service --target-port=8080
+
+````
+** NOTE : Notice the port numbers we are specifying here. They match the port numbers we used in the loopback4 application.
```

Comment:
Please use "note" or "tip" alert here.

See https://loopback.io/doc/en/contrib/alerts.html#types-of-alerts

Reply:
done